common/djangoapps/student/helpers.py

```diff
@@ -98,6 +105,9 @@ def check_verify_status_by_course(user, course_enrollments):
 
             relevant_verification = SoftwareSecurePhotoVerification.verification_for_datetime(deadline, verifications)
 
+            verification_expiring_soon = SoftwareSecurePhotoVerification.is_verification_expiring_soon(
```

Comment:
Why is this value assigned inside the loop? It doesn't seem to have anything to do with the enrollment.